scripts/ironic-probe.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+# shellcheck disable=SC1091
+. /bin/ironic-common.sh
+# shellcheck disable=SC1091
+. /bin/auth-common.sh
+
+PROBE_CURL_ARGS=
+if [[ "${IRONIC_REVERSE_PROXY_SETUP}" == "true" ]]; then
+    if [[ "${IRONIC_PRIVATE_PORT}" == "unix" ]]; then
```

Comment:
I guess this assumes that probes are running with the same variables as the container itself?

Reply:
Yes, this is the same logic we used before I just lifted it from the config script and placed here with small modification.